SlidingUpPanel.js

```diff
@@ -414,7 +418,8 @@ class SlidingUpPanel extends React.PureComponent {
     const animatedContainerStyles = [
       styles.animatedContainer,
       transform,
-      {height, bottom: -height}
+      {height, bottom: -height},
+      this.props.contentStyle && this.props.contentStyle
```

Comment:
```suggestion
      this.props.contentStyle
```
No need to check for the existence.